README.md

```diff
@@ -41,3 +41,9 @@ To get a unit test coverage report, you can run `sbt clean coverage test coverag
 then open the resulting coverage report `target/scala-2.11/scoverage-report/index.html` in a web browser.
 
 Test coverage threshold is set at 85% - so if you commit any significant amount of implementation code without writing tests, you can expect the build to fail.
+
+## All tests and checks
+
+This is an sbt command alias specific to this project. It will run a scala style check, run unit tests, run integration tests and produce a coverage report.
```

Comment:
Could you please replace an with a (in '**This is an sbt command alias specific to this project**')?

Also please break the line after the word integration (to make it within 120 characters limit) as this has become a very long line and better to keep the colon (:) instead of period(.) at the end of the statement